opensquirrel/decomposer/aba_decomposer.py

```diff
@@ -31,6 +31,19 @@ def __init__(self) -> None:
         self.index_a = self._gate_list.index(self.ra)
         self.index_b = self._gate_list.index(self.rb)
 
+    @staticmethod
+    def _find_non_used_index(axis_list: list[int]) -> int:
+        """Finds the index of the axis object that is not used in the decomposition.
+        For example, if one selects the ZYZ decomposition, the integer returned will be 0 (since it is X).
+
+        Parameters:
+            axis_list: list of the axis indices in use
+
+        Returns:
+            Index of the axis object that is not used in the decomposition.
+        """
+        return sorted(set(range(0, 3)) - set(axis_list))[0]
```

Comment:
```suggestion
    def _find_non_used_index(self) -> int:
        """Finds the index of the axis object that is not used in the decomposition.
        For example, if one selects the ZYZ decomposition, the integer returned will be 0 (since it is X).

        Returns:
            Index of the axis object that is not used in the decomposition.
        """
        return ({0,1,2} - {self.index_a, self.index_b}).pop()
```
The method acts on `self.index_a` and `self.index_b` so should probability not be a `staticmethod`.